### PR TITLE
Spawn multiple applications on one group

### PR DESCRIPTION
**Pull request by apinsard**

### CODE REVIEW

File: libqtile/dgroups.py [hunk 1]
```diff
@@ -127,8 +127,13 @@ def _setup_groups(self):
             self.add_dgroup(group, group.init)
 
             if group.spawn and not self.qtile.no_spawn:
-                pid = self.qtile.cmd_spawn(group.spawn)
-                self.add_rule(Rule(Match(net_wm_pid=[pid]), group.name))
+                if isinstance(group.spawn, str):
```
Comment (tych0) [inline libqtile/dgroups.py]: You may want to use `six.string_types` here instead of `str` to be compatible with py2/3.

Comment (apinsard) [inline libqtile/dgroups.py]: Right, fixed.

**Comment by Roger** [comment]
Would you mind updating doc/type here? https://github.com/qtile/qtile/blob/develop/libqtile/config.py#L437

**Comment by apinsard** [comment]
Sure. I don't know this doc format, may I set type as "string or iterable" or "string or strings list"?

**Comment by Roger** [comment]
Looking at sphinx docs i guess "string or list of strings" is ok.

**Comment by flacjacket** [comment]
Thanks for this!

**Pull request merged**